main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
"rarely happen" i mean failed to delete file other than ".git", just in case  if there is other file might failed to be deleted. 

i am trying now, should have update soon.